user-org-service/app/controllers/logsmanager/validator/LogValidator.java

```diff
@@ -0,0 +1,44 @@
+package controllers.logsmanager.validator;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.sunbird.util.LoggerEnum;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.UserOrgJsonKey;
+import org.sunbird.util.request.Request;
+
+public class LogValidator {
```

Comment:
Like this class, can we keep our own logger class and have all methods static, like startTrace, endTrace and log in it? I'm still not fully convinced if there is any good use on ProjectLogger. Just asking for ideas.

Reply:
Sir, 
ProjectLogger class has already all static methods implemented and what i think that ProjectLogger class in future will be needed everywhere so it should present inside util.
if we make another class in user-org-service then we also need to make same other class in user-service org-service where we will have our  business logic. plz correct me if i am wrong